src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise is is created!');
+  });
+
+  document.addEventListener('click', () => {
+    reject(Error);
+  });
+});
+
+function getMassage(message, classMassage) {
+  const div = document.createElement('div');
+  const body = document.querySelector('body');
+
+  div.innerText = message;
+  div.className = classMassage;
+  body.lastElementChild.before(div);
+};
+
+promise1.then(resolve => {
```

Comment:
the callback accepts anything you passed to `resolve` in your promise. In this case, it's a message, so variable name should be `message`, etc

```suggestion
promise1.then(message => {
```